app/factory/Layer.js

```diff
@@ -107,6 +114,87 @@ Ext.define('CpsiMapview.factory.Layer', {
         return mapLayer;
     },
 
+    /**
+     * Loops through all layers, identifies switch layers
+     * and replaces them if required
+     *
+     * @param {ol.Object.Event} evt The event which contains the view.
+     */
+    handleSwitchLayerOnResolutionChange: function (evt) {
+
+        var resolution = evt.target.getResolution();
+
+        // TODO: this needs to be called more elegantly
+        var allLayers = BasiGX.util.Map.getMapComponent().getMap().getLayers();
+        var overlayGroup =  BasiGX.util.Layer.getLayerByName('Layers', allLayers).getLayers();
```

Comment:
Is there a better to way to call the overlay group?

Reply:
At the moment there is no better way. Maybe we should introduce a util class to query the layer groups in the future.